next/13/app-directory-boilerplate/index.ts

```diff
@@ -1016,12 +1035,17 @@ const handleData: HandleData<Dependencies, State> = async (
 			return buildPageFileData(api, path, options, filePurpose);
 		}
 
-		if (
-			filePurpose === FilePurpose.ROOT_LAYOUT &&
-			options.underscoreDocumentData
-		) {
+		if (filePurpose === FilePurpose.ROOT_LAYOUT) {
 			const { tsmorph } = api.getDependencies();
 
+			if (options.underscoreDocumentData === undefined) {
```

Comment:
`underscoreDocumentData` is string or null, no?

Reply:
its string or undefined